src/drive/share/index.ts

```diff
@@ -569,6 +570,18 @@ export class Share {
     return this.client.get(`sharings/${itemType}/${itemId}/type`, headers);
   }
 
+  /**
+   * Get Sharing information
```

Comment:
Perhaps we could be a bit more specific with the description: 
`Obtains detailed sharing information for a given file or folder`